src/lib/layers/editable-geojson-layer.js

```diff
@@ -575,6 +580,20 @@ export default class EditableGeoJsonLayer extends EditableLayer {
     }
   }
 
+  isPointerWithinHintPointMaxDistance(screenCoords: Array<any>, snapPoint: Object) {
+    const { hintPointMaxDistance } = this.props;
+    let isWithinMaxDistance = true;
+    if (hintPointMaxDistance) {
+      const snapPointScreenCoords = this.context.viewport.project(snapPoint.geometry.coordinates);
+      const distanceFromPointerToSnapPointPixels = math.distance(
```

Comment:
I don't think we need to bring in a whole new dependency just to do distance. Should be able to simply do:
```
side1 = abs(screenCoords[0]-snapPointScreenCoords[0])
side2 = abs(screenCoords[1]-snapPointScreenCoords[1])
distance = sqrt(side1^2+side2^2)
```